python_ta/patches/error_messages.py

```diff
@@ -4,18 +4,193 @@
+            {'R0123': 'tHE \'is\' keyword compares two objects location in memory rather '
+                      'than value equality. When performing comparison with a literal, '
+                      'this is usually not what you want. Did you mean \'==\'?',
+             'C0123': 'You should use \'isinstance\' rather than \'type\' to '
+                      'perform a typecheck.'
+             },
+        'NameChecker':
+            {'C0102': 'Black listed name "%s". You should give your variables '
+                      'meaningful rather than generic names.',
+             'C0103': '%s name "%s" doesn\'t conform to %s. '
+                      'Please refer to the PyTA Help Documentation for more information '
+                      'on Python Naming Conventions.'
+             },
         'PassChecker':
-             {'W0107': 'Unnecessary pass statement (you should remove this)'}
+            {'W0107': 'Unnecessary pass statement (you should remove this). '
+                      'A pass statement should only be used to represent '
+                      'an \"empty\" code block, where nothing happens.'},
+        'BasicErrorChecker':
+            {'E0101': '__init__ is called by the special method __new__ '
+                      'when a new object is instantiated, and will raise '
+                      'a TypeError if __init__ returns anything other than None.',
+             'E0102': '%s already defined line %s. '
+                      'You should make sure all functions, methods and classes '
+                      'defined have different names.',
+             'E0103': 'The %r keyword should only be used inside a loop.',
+             'E0104': 'The \'return\' keyword is used to end the execution of a function '
+                      'and return a result, so you should only use it inside of a function. '
+                      'Note that if no return statement is written, Python implicitly '
+                      'ends the execution of a function with return None',
+             'E0108': 'Duplicate parameter name %s in function definition. '
+             }
     },
     'pylint.checkers.refactoring': {
         'RefactoringChecker':
-             {'R1710': 'Since this function can return a non-None value, you should '
-                       'explicitly write "return None" whenever None is returned by '
-                       'this function. (Possibly including possibly at the end of the function body.)'
+            {'R1710': 'Since this function can return a non-None value, you should '
+                      'explicitly write "return None" whenever None is returned by '
+                      'this function. (Possibly including at the end of the function body.)'
              }
     },
     'pylint.checkers.design_analysis': {
-        'MisdesignChecker': {'R0913': 'Too many parameters (%s/%s)'}
+        'MisdesignChecker': {'R0912': 'The function has too many branches (%s out of a default limit '
```

Comment:
Remove the word "default" (since these are configurable), and rephrase to "(%s, exceeding limit %s)". Applies for similar messages below.

Reply:
Also, use "This" instead of "the" (throughout this section's messages)